site/_data/i18n/fugu_showcase.yml

```diff
@@ -4,6 +4,9 @@ select_api:
 search_apps:
   en: Search apps
 
+rss_feed:
+  en: RSS feed
```

Comment:
It's useful enough to have it in common.yaml

Reply:
Done in https://github.com/GoogleChrome/developer.chrome.com/pull/5262/commits/22b1fa01bb5955b73185c4e3e8c9c8537f888897.